File: Codeforces Contests/19/1.cpp
#include<bits/stdc++.h>
using namespace std;
#define int long long
#define fastio ios_base::sync_with_stdio(false);cin.tie(NULL);cout.tie(NULL);
int32_t main(){
    fastio;
    int t;
    cin>>t;
    while(t--){
        int n;
        cin>>n;
        
        vector<int> nums(n);
        for(int i=0;i<n;i++)
            cin>>nums[i];
        
        vector<int> tmp=nums;
        sort(tmp.begin(),tmp.end());
        int ans=0;
        for(int i=0; i<n; i++) {
            if(tmp[i]!=nums[i]){
                cout<<"YES"<<endl;
                ans=1;
                break;
            }
        }
        if(ans==0)
            cout<<"NO"<<endl;
    }
}

File: Codechef Contests/LT/1.py
for i in range(int(input())):
    n=input()
    ans=False
    for i in range(1, len(n)):
        if((int(n[:i])+int(n[i:]))%2==0):
            ans=True
            break
    
    if(ans):
        print("YES")
    else:
        print("NO")